app/react/Attachments/components/AttachmentForm.js

```diff
@@ -4,17 +4,17 @@ import { connect } from 'react-redux';
 import { Form, Field } from 'react-redux-form';
 
 import { FormGroup, Select } from 'app/ReactReduxForms';
-import { elasticLanguages } from 'shared/languagesList';
+import { elasticLanguages } from 'shared/language';
 import t from 'app/I18N/t';
 import ShowIf from 'app/App/ShowIf';
 
 export class AttachmentForm extends Component {
   render() {
     const { model } = this.props;
     const validators = { originalname: { required: val => !!val && val.trim() !== '' } };
-    const languageOptions = Object.keys(elasticLanguages).map(key => ({
-      value: elasticLanguages[key].franc,
-      label: elasticLanguages[key].elastic,
+    const languageOptions = elasticLanguages.map(item => ({
```

Comment:
Our code base usually either uses a descriptive name of what the iterable `item` is, or, in case of very short and "obvious" maps, we use a shorthand letter, as every iterable thing is an "item".  So, either:
`elasticLanguages.map(language => ({`
or something similar... or maybe directly destructure in the function call?
`elasticLanguages.map(({ISO639_3, elastic}) => ({value: ISO639_3, label: elastic}))`

Something more semantic than `item`?

Reply:
Got it!